app/scripts/lib/channels/notifier.js

```diff
@@ -20,12 +20,12 @@ define(function (require, exports, module) {
     CHANGE_PASSWORD: {
       name: 'fxaccounts:change_password',
       schema: {
-        email: 'String',
-        keyFetchToken: 'String',
-        sessionToken: 'String',
-        uid: 'String',
-        unwrapBKey: 'String',
-        verified: 'Boolean'
+        email: Vat.email().required(),
+        keyFetchToken: Vat.keyFetchToken().optional(),
```

Comment:
I had them as required to begin with and the functional tests failed - neither keyFetchToken nor unwrapBKey was being passed. I am confused what changed, whether the notifications weren't being sent based on whether the data was considered valid, or whether there was an error in the validation. My guess is the former. I'll dig in some more.